students/srepking/Lesson04/mailroom2.py

```diff
@@ -96,16 +102,18 @@ def letters_for_all():
             to_file.write(letter)
 
 
-def goodbye():
+def wrong_choice():
     pass
 
+def quit():
+    exit()
```

Comment:
`quit` is a built-in function (which is why it's highlighted in blue vs other functions you have).

you want to avoid over-writing built-ins 😉; rename it to something like `quit_program`